spec/src/main/asciidoc/core/events.asciidoc

```diff
@@ -296,22 +296,27 @@ An event parameter may have multiple qualifiers.
 public void afterDocumentUpdatedByAdmin(@Observes @Updated @ByAdmin Document doc) { ... }
 ----
 
-Then this observer method will only be notified if all the observed event qualifiers are specified when the event is fired:
+Then this observer method will be notified if the set of observer qualifiers is a subset of the fired event's qualifiers:
 
 [source, java]
 ----
 @Inject Event<Document> documentEvent;
 ...
-documentEvent.select(new UpdatedQualifier(), new ByAdminQualifier()).fire(document);
+documentEvent.select(new UpdatedQualifier(), new ByAdminQualifier(), new ClarificationQualifier()).fire(document);
 ----
 
-Other, less specific, observers will also be notified of this event:
+In the above example the event is fired with `@ByAdmin`, `@Updated`, `@Clarification` and `@Any` qualifiers. `@Any` is added automatically as described in <<event_types_and_qualifier_types>>. The observer qualifiers are `@Updated`, `@ByAdmin` and again `@Any`.
+Observer qualifiers therefore form a subset of event qualifiers and the observer will be notified.
+
+Other, less specific, observer will also be notified of this event:
 
 [source, java]
 ----
 public void afterDocumentUpdated(@Observes @Updated Document doc) { ... }
 ----
 
+Last but not least, the following observer is the most general one and since it does not declare any qualifier, it has `@Any` by default.
```

Comment:
@manovotn There is no @Any added on observer side. IMO, the fact this observer gets notified come from section [10.3 Observer resolution](http://docs.jboss.org/cdi/spec/1.2/cdi-spec.html#observer_resolution)

> An event is delivered to an observer method if:
> 
> * The observer method belongs to an enabled bean.
> * An event type is assignable to the observed event type, taking type parameters into consideration.
> * **The observer method has no event qualifiers** or **has a subset of the event qualifiers.**
> An observer method has an event qualifier if it has an observed event qualifier with (a) the same type and (b) the same annotation member value for each member which is not annotated `@javax.enterprise.util.Nonbinding`.
>...

I agree that empty set is a subset of all sets but "Average Joe" needed this Captain Obvious clarification to make the difference with [Typesafe Resolution](http://docs.jboss.org/cdi/spec/1.2/cdi-spec.html#performing_typesafe_resolution) for bean injection where there is never an empty set of qualifier (empty set is replaced by `@Default`):

> A bean is _assignable_ to a given injection point if:
> 
> * ...
> * The bean has all the required qualifiers. **If no required qualifiers were explicitly specified, the container assumes the required qualifier `@Default`**. A bean has a required qualifier if it has a qualifier with (a) the same type and (b) the same annotation member value for each member which is not annotated `@javax.enterprise.util.Nonbinding`.